src/otaclient/client_package.py

```diff
@@ -325,20 +326,14 @@ def __init__(
 
     def _cleanup_mount_point(self) -> None:
         """Cleanup the mount point."""
-        try:
-            # unmount the dynamic client mount point
-            cmdhelper.ensure_mointpoint(self._mount_base, ignore_error=True)
-            cmdhelper.ensure_umount(
-                self._mount_base, ignore_error=False, max_retry=0, retry_interval=0
-            )
-        except Exception as e:
-            logger.warning(f"error while unmounting dynamic client mount point: {e}")
+        # unmount the dynamic client mount point
+        cmdhelper.ensure_mointpoint(self._mount_base, ignore_error=True)
+        cmdhelper.ensure_umount(
+            self._mount_base, ignore_error=True, max_retry=0, retry_interval=0
+        )
 
-        try:
-            # remove the dynamic client mount point
-            shutil.rmtree(self._mount_base, ignore_errors=False)
-        except Exception as e:
-            logger.warning(f"error while removing dynamic client mount point: {e}")
+        # remove the dynamic client mount point
+        shutil.rmtree(self._mount_base, ignore_errors=True)
```

Comment:
Exceptions are ignored in the original code, so using ignore_error(s) is sufficient.